modules/infrastructure-moving-logging.adoc

```diff
@@ -7,6 +7,6 @@
 
 [IMPORTANT]
 ====
-The log aggregation solution in {product-title} is not installed by default and
-cannot currently be deployed.
-====
\ No newline at end of file
+The log aggregation solution in {product-title} is not installed by default.
```

Comment:
Sorry. Should have seen this earlier. The _log aggregation solution_ is now called _cluster logging_.

Reply:
Now is plenty soon enough. :) Do you see anything else that needs to change immediately? If not, I'll merge.

Reply:
Merge away. Let's at least correct the initial issue. Shall I update that Community Discussions issue?

Reply:
Please do!